go/libkb/constants.go

```diff
@@ -41,12 +41,26 @@ const (
 	NoRunMode         RunMode = kbconst.NoRunMode
 )
 
-var ServerLookup = map[RunMode]string{
-	DevelRunMode:      DevelServerURI,
-	StagingRunMode:    StagingServerURI,
-	ProductionRunMode: ProductionServerURI,
+func ServerLookup(env *Env, mode RunMode) (string, error) {
```

Comment:
Can we move this function out of constants?  Trying to keep it "constants" only.

Reply:
Done. Moved to `client.go` since that seems like a reasonable place for it.